src/planner/linux/steam_deck.rs

```diff
@@ -247,3 +252,9 @@ impl Into<BuiltinPlanner> for SteamDeck {
         BuiltinPlanner::SteamDeck(self)
     }
 }
+
+#[derive(thiserror::Error, Debug)]
+enum SteamDeckError {
+    #[error("`{0}` is not an absolute path, bind mounts require an absolute path and it cannot be canonicalized during planning")]
```

Comment:
"and it cannot be canonicalized during planning" just seems weird to me -- does that give the user any valuable information? I think lopping that part of would be just fine, IMHO.

Reply:
Mmm I reworded it to `#[error("`{0}` is not an path that can be canonicalized into an absolute path, bind mounts require an absolute path")]` as I believe the user does want to know it was a canonicalization issue

Reply:
I think that's OK!